logidze.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |spec|
   rails_version = ">= 6.0"
   spec.add_dependency "railties", rails_version
   spec.add_dependency "activerecord", rails_version
+  spec.add_dependency "concurrent-ruby", "1.3.4"
```

Comment:
Let's make a development dependency for now; we shouldn't freeze indirect dependencies for end users, and for us that would be enough.

And could you please add a comment with the link to the issue? (So we can drop it as it's resolved)

Reply:
will do!